src/riot-rs-threads/src/lib.rs

```diff
@@ -247,14 +244,6 @@ pub unsafe fn thread_create_raw(
     })
 }
 
-/// Returns the [`ThreadState`] for this `thread_id`.
-///
-/// Returns `None` if `thread_id` is out of bound or no thread with
-/// valid state exists.
-pub fn get_state(thread_id: ThreadId) -> Option<ThreadState> {
-    THREADS.with(|threads| threads.get_state(thread_id))
-}
-
```

Comment:
Removed because it's not safe to expose `ThreadState`. The variants `ChannelRxBlocked(usize)` and `ChannelTxBlocked(usize)` include a pointer to the location from/ to where read/ write, which a user shouldn't access.